components/statuslight/stories/template.js

```diff
@@ -27,12 +27,10 @@ export const Template = ({
 	</div>
 `;
 
-// TODO: the accent variant will be removed in S2.
```

Comment:
I put this `TODO` in `main` during the [status light docs migration](https://github.com/adobe/spectrum-css/pull/3152/files), but during the [S2 status light migration](https://github.com/adobe/spectrum-css/pull/2818/files) I had removed the accent variant already.